redis/src/test/scala/zio/redis/RespArgumentSpec.scala

```diff
@@ -0,0 +1,23 @@
+package zio.redis
+
+import zio.test._
+
+object RespArgumentSpec extends BaseSpec {
+  def spec: Spec[Any, RedisError.ProtocolError] =
+    suite("RespArgument")(
+      suite("BulkString.asCRC16")(
+        test("key without braces") {
+          val key = RespArgument.Key("hello world")
+          assertTrue(15332 == key.asCRC16)
```

Comment:
Nitpick (I'm fine with leaving it as it is as well):

```suggestion
          assertTrue(15332 == RespArgument.Key("hello world").asCRC16)
```